layouts/api/argument_list.ejs

```diff
@@ -1 +1,5 @@
-<span class="api-entry__argument-list">(<%- item.arguments?.map(a => `<span class="api-entry__argument">${a.name}</span>`)?.join(', ') ?? '' -%>)</span>
+<span class="code-doc__entry__argument-list">
+  (
+    <%- item.arguments?.map(a => `<span class="code-doc__entry__argument">${a.name}</span>`)?.join(', ') ?? '' -%>
+  )
+</span>
```

Comment:
Have you tested to ensure this won't have side effects? Seems like it'll introduce whitespace where we don't want it.

Reply:
I didn't see any extra whitespace, but I'll go ahead and double check on that one, just spaced it out for easier reading without realizing the whitespace was a concern. If it is I'll revert it

Reply:
Great call here. This does in fact introduce unwanted whitespace. I'll change it back.